### PR TITLE
[freeboxos] Align audio sink to freebox binding

### PR DESCRIPTION
**Pull request by lolodomo**
Related to #15113

Signed-off-by: Laurent Garnier <lg.hc@free.frf>

### CODE REVIEW

**Review by clinique — COMMENTED** [review]
LGTM

**Review by clinique — APPROVED** [review]

File: bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/handler/AirMediaSink.java [hunk 1]
```diff
@@ -111,46 +113,69 @@ public String getId() {
...
-                streamServed.playEnd().thenRun(() -> this.playbackFinished(audioStream));
-                logger.debug("AirPlay audio sink: process url {}", callbackUrl + streamServed.url());
-                playMedia(manager, callbackUrl + streamServed.url());
-            } catch (FreeboxException e) {
-                logger.warn("Audio stream playback failed: {}", e.getMessage());
+                streamServed = audioHTTPServer.serve(audioStream, 5, true);
+            } catch (IOException e) {
+                tryClose(audioStream);
+                throw new UnsupportedAudioStreamException(
+                        "AirPlay device was not able to handle the audio stream (cache on disk failed).",
```
Comment (jlaur) [inline bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/handler/AirMediaSink.java:140]: AirPlay intended? Just checking.

**Review by jlaur — APPROVED** [review]
LGTM, just one question since I'm not into Freebox, and don't know if Apple-related.

File: bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/handler/AirMediaSink.java (hunk 1 above)
Comment (lolodomo) [reply bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/handler/AirMediaSink.java:140]: Yes, it concerns Freebox devices (Freebox players) having AirPlay support.

**Pull request merged**